compiler.rst

```diff
@@ -10,16 +10,16 @@ Abstract
 Historically (through 2.4), compilation from source code to bytecode
 involved two steps:
 
-1. Parse the source code into a parse tree (Parser/pgen.c)
-2. Emit bytecode based on the parse tree (Python/compile.c)
+1. Parse the source code into a parse tree (:file:`Parser/pgen.c`)
+2. Emit bytecode based on the parse tree (:file:`Python/compile.c`)
```

Comment:
Yes.  We have dropped almost all references to 2.x from 3.x docs (but the devguide is not versioned) and Nick confirmed my suspicion that a 2.4 reference is no longer needed for 2.x either.  Not knowing the history, I found the current text it initially confusing.